docs/source/testing/larva/ignores.rst

```diff
@@ -0,0 +1,284 @@
+Ignores
+========
+
+When creating Larva tests, you will find that some constantly changing values will make the tests fail at certain points because there is a difference between the current result of the test and the expected result of the test, for example, the “timestamp”. After all, time changes every second. For solving this kind of issue, Larva has several ways for ignoring parts of messages to compare them. This subsection is intended to name and describe all those ignores within the Larva testing tool.
+
```

Comment:
It is good that you give the overview right at the top of this section, because experienced Frank developers can now quickly find the ignore option they want. Newbies can better read the examples of ignores first before turning to the overview. Can you add one or two sentences to guide newbies in reading the text the right way?